build-integrations/global/plugins.sbt

```diff
@@ -23,9 +32,19 @@ libraryDependencies := {
   if (sbtVersion.startsWith("1.")) {
     List(
       sbtPluginExtra("com.lucidchart" % "sbt-scalafmt" % "1.15", sbtVersion, scalaVersion),
-      sbtPluginExtra("com.eed3si9n" % "sbt-assembly" % "0.14.6", sbtVersion, scalaVersion)
+      sbtPluginExtra("com.eed3si9n" % "sbt-assembly" % "0.14.6", sbtVersion, scalaVersion),
+      typesafeConfig,
+      metaconfigCore,
+      metaconfigDocs,
+      metaconfigConfig,
+      circeDerivation
     )
   } else {
-    List()
+    List(
+      typesafeConfig,
```

Comment:
We need to do this for all the builds using 2.10 (sbt 0.13) because circe-derivation and metaconfig are not published to 2.10. As we only care about the encoders, we add circe core and circe generic and make the module cross-compile.